opus/application/static_media/js/hash.js

```diff
@@ -47,14 +47,19 @@ var o_hash = {
                         // opus.prefs.view = search, browse, cart, or detail
                         // opus.prefs.browse =  'gallery' or 'dataTable',
                         let page = o_browse.getCurrentPage();
-                        hash.push("page=" + page);
+                        // we will no longer have page in url, so no need to push page
+                        // hash.push("page=" + page);
+                        break;
+                    case "startobs":
+                        hash.push("startobs=" + opus.currentObs);
```

Comment:
Likewise we should comment out this code (and the definition of currentObs) until merged with the startobs branch.